components/formats-gpl/src/loci/formats/in/TargaReader.java

```diff
@@ -85,12 +85,11 @@ public TargaReader() {
     options.maxBytes = planeSize;
     options.bitsPerSample = bits;
 
-    RandomAccessInputStream s = in;
-
     if (compressed) {
```

Comment:
The changes to this reader are resulting in test failures for testPixelsHashes. See https://web-proxy.openmicroscopy.org/east-ci/job/BIOFORMATS-test-folder/43281/console